reformat.sh

```diff
@@ -15,4 +15,4 @@
 
 set -euxo pipefail
 
-find {.,aws-encryption-sdk-cpp}/{include,source,tests} examples -name '*.h' -or -name '*.c' -or -name '*.cpp' | xargs clang-format -i
+find {.,aws-encryption-sdk-cpp}/{include,source,tests} .cbmc-batch examples -name '*.h' -or -name '*.c' -or -name '*.cpp' | xargs clang-format -i
```

Comment:
This line makes ```reformat.sh``` check the ```cbmc-batch``` folder.  On aws-c-common, this is checked in CI - not sure if this is also true for ESDK.

Reply:
@karlw00t re your comment.  Do you know if reformat.sh runs on CI?